### ISSUE TITLE
file_path cop seems to produce wrong snake_case 

### ISSUE THREAD

**Issue by Jack12816** [backus/rubocop-rspec#227]
The current ["camel case to underscore" logic](https://github.com/backus/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/file_path.rb#L90-L94) of the file_path cop produces for "S3MailRoom" -> "s_3_mail_room*_spec.rb", which is quite strange. 

[Rails/ActiveSupport](http://api.rubyonrails.org/classes/ActiveSupport/Inflector.html#method-i-underscore) produce "s3_mail_room" instead.
```
irb(main):007:0> include ActiveSupport::Inflector
irb(main):006:0> underscore('S3MailRoom')
=> "s3_mail_room"
```

I think is is a bug on the cop, if the issue is confirmed I can fit it with a PR :)

**Comment by backus** [comment]
I think @bquorning fixed this in #195 but we haven't released it yet. Can you try master and see if it works?

**Comment by Jack12816** [comment]
Oh great, this is fixed on master! :+1: Instead I have now 224 new offenses, but hey ^__^
